app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="<%= dom_id group %>" class="group-card">
+  <a href = <%= group_entities_url(group.id) %> class="group-card-link">
+    <p class="group-card-img">
+      <%= image_tag url_for(group.avatar) %>
+    </p>
+
+    <p class="group-card-name">
+      <%= group.name %>
+    </p>
+
+    <p class="group-card-amount">
+      $<%= group.transactions_sum %>
+    </p>
+  </a>
```

Comment:
- [OPTIONAL] The card should have a category image, name, `created date`, and amount. kindly let's display the creation date for each category 👇🏻  


Original Design
:------:
 ![image](https://user-images.githubusercontent.com/79058364/231509615-c78c3511-faca-401e-a7a5-82cc0560550a.png)